CONTENT/about.markdown

```diff
@@ -8,42 +8,49 @@
 Our mission is to empower kids with programming skills for making social change!
 
 With wizard amigos workshops
-Kids will:
-- ...
-Tech Companies will:
-- ...
-Mentors will:
-- ...
+Kids will be able to:
+- get to know basic programming concepts and develop logical thinking
+- learn fastest growing programming language - JavaScript and its ecosystem
+- learn to collaborate with other learners
+- find out everything about how to apply her/his new skills with learning-by-doing approach
+
+Tech Companies will be able to:
+- identify and access market opportunities for educational applications and classroom integration of new products.
+
+Mentors will be able to:
+- continue learning 
+- get paid for sharing their skills with kids
+- be part of community of continuous learners and get help with getting their projects off the ground
+
 ## Our story
-Jan 2014 we started Coding Amigos Hackathons, we organized over 70 events and co-learning workshops. Before that we participated in several groups- from universities, incubators, rails girls, startup weekends, open tech school, ...
+In early 2014 we started Coding Amigos Hackathons in Berlin. We organized over 70 events and co-learning workshops. Before that we participated in several groups: University Incubator, Social Media Week, Rails Girls, StartUp weekend, Open Tech School etc.
 
-During last year,we have met a lot of coding learners and got to know several techniques for beginners and advanced learners.
+During the last year we have met a lot of coding learners and got to know several techniques for beginners and advanced learners. We realised we have enough experiences to bring this skills also to the youngest segment. And so the idea for programming school for kids was born.
 
 Programming is a new literacy
 
 Wizard Amigos Institute has opened its doors in May 2015.
 
 ## CURRENT TEAM
-* [Nina Breznik - [@ninabreznik](#twitter)](#github)
-* [Alexander Praetorius - [@serapath](#twitter)](#github)
-
+### [Nina Breznik - [@ninabreznik](https://twitter.com/ninabreznik)](https://github.com/ninabreznik)
+- co-organizer of [Coding amigos](http://www.meetup.com/CodingAmigos/)
+- founder of [Zweidesign](http://zweidesign.co/)
+- [Rails Girl](http://railsgirlsberlin.de/)
 
-https://gitter.im/wizardamigosinstitute/chat
-wizard@amigos.institute
-http://wizard.amigos.institute/
+### [Alexander Praetorius - [@serapath](https://twitter.com/serapath)](https://github.com/serapath)
+- co-organizer of [Coding amigos](http://www.meetup.com/CodingAmigos/)
+- co-founder of [Frappe](http://frappe-rheinmain.de/)
+- open source contributor to [Atomify](https://github.com/atomify/atomify)
 
 If you want to know more, please checkout
 [more details](https://github.com/wizardamigosinstitute/organisation/blob/master/README.md)
 
 ### Follow Us
-* Google Plus
-* Twitter
-* Facebook
-* Mailchimp (subscribe)
```

Comment:
It makes sense to add Mailchimp. I will do that. But for Google plus I don't know. Usually they say - use 2 or 3 channels for your business that you know how they work and that you actually maintain. To have it just because doesn't really make sense, but maybe we can try and if lots of people join to Google page, then we can do more work there. Could you do Google page and add link and I will do Mailchimp?